opentreemap/treemap/views/tree.py

```diff
@@ -118,6 +120,49 @@ def search_tree_benefits(request, instance):
     return context
 
 
+def simplified_tree_benefits(request, instance):
+    unit = 'kg/rok'
```

Comment:
Should it be hardcoded? What about possible translations in the future?

Reply:
In fact this fork is merely a proof of concept for the Polish map, so there is no use in maintaining translations. It's also extremely complicated to manage translations in this project, I hope you remember. :D